core.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/impl/locks/CASLockFactoryBuilder.java

```diff
@@ -130,8 +112,8 @@ public final ConsistencyType getConsistencyType()
         return myConsistencyType;
     }
 
-    public final Node getNode()
+    public final UUID getHostId()
```

Comment:
This is actually making host id static, it should not me static, we don't have anymore a local node, so it should be a method param